src/components/HomeSection.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import HomeSection from './HomeSection';
+
+describe('HomeSection', () => {
+  test('renders HomeSection component', () => {
```

Comment:
This test renders a `HomeSection` but doesn't test anything about it. You probably want to have multiple tests that try out the default property values and varying values and makes sure that:
1. They render and don't break unexpectedly
2. The properties given are reflected in the rendered DOM (e.g. the title is there, the number of buttons given are there, etc.) 

I suggest reading the [Testing Library Docs](https://testing-library.com/docs/) to get a better sense of what to test. Specifically, you'll want to learn about [Queries](https://testing-library.com/docs/queries/about/) and read about the [DOM Testing Library](https://testing-library.com/docs/dom-testing-library/intro) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro). There are plenty of examples there as well as in other `.test.jsx` files in our repo.